net/proxy_resolution/proxy_config_service_tor_unittest.cc

```diff
@@ -0,0 +1,146 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/net/proxy_resolution/proxy_config_service_tor.h"
+
+#include <string>
+
+#include "base/macros.h"
+#include "net/proxy_resolution/proxy_config_with_annotation.h"
+#include "net/test/test_with_scoped_task_environment.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "url/gurl.h"
+
+namespace net {
+
+class ProxyConfigServiceTorTest : public TestWithScopedTaskEnvironment {
```

Comment:
Do we really need `TestWithScopedTaskEnvironment`?

Reply:
I haven't even looked at these tests and I'm not sure if they even compile or not anymore with my changes

Reply:
@darkdh is there a reason for TestWithScopedTaskEnvironment?